book/src/api-vc-auth-header.md

```diff
@@ -6,13 +6,13 @@ The validator client HTTP server requires that all requests have the following
 HTTP header:
 
 - Name: `Authorization`
-- Value: `Bearer <api-token>`
+- Value: `Bearer <string>`
 
-Where `<api-token>` is a string that can be obtained from the validator client
-host. Here is an example `Authorization` header:
+Where `<api-token>` is a string of 33 characters that can be obtained from the validator client
```

Comment:
```suggestion
Where `<api-token>` is a string that can be obtained from the validator client
```

The token may not be 33 characters if it is initialised by the user, or still using the old format. I don't think we need to mention this

Reply:
May I know how to still load the old format of api-token?

Reply:
The token is just whatever is in the `api-token.txt` file. So if it is the old-style token like `api-token-$blah` then we will just use those bytes as the token

Reply:
ok, so if the `validators` folder are deleted and users are using 5.3.0 and above, it will load the new api-token? (the string, not the `api-token-longstring`